data/remnant missions.txt

```diff
@@ -1252,3 +1252,237 @@ mission "Remnant: Expanded Horizons Quarg 3"
+			`	"Our thanks, Captain <last>. Having a partially functional drive like this will be very beneficial to our efforts." His voice cracks a bit, as if unused to being used much.`
+			choice
+				`	"What are you studying?"`
+					goto studying
+				`	"Great! I will find more."`
+					end
+			label studying
+			`	"Our previous research into Jump Drives determined that these drives are unsurprisingly multi-part systems. We know the outer part that we can see and handle is mostly the machinery for generating some kind of containment field or pocket. As to how it works or why it is needed..." his chant trails off and he makes a gesture you suspect may be the equivalent of a shrug. "Anyway, our team is focused on the nature of this outer system. It is our hope that - damaged as it is - its continued functionality might allow us to look a little farther inside its inner workings to figure out exactly what that field does and how it is used."`
+			`	"Well, my compatriots finished unloading rather quickly, so I should be off. For all our sakes, I hope you find more of these for us."`
+		outfit "Jump Drive (Broken)" -1
+		"remnant: broken jump drive count" ++
+		payment 500000
+		"reputation: Remnant" ++
+
+
+
+mission "Remnant: Broken Jump Drive 3"
+	name "Broken Jump Drive Delivery"
+	description "The researchers on <planet> need a broken jump drive for testing."
+	to offer
+		has "Remnant: Broken Jump Drive 2: done"
+	source
+		government "Remnant"
+	destination
+		government "Remnant"
+	on offer
+		require "Jump Drive (Broken)"
+		conversation
+			`As you arrive in the spaceport concourse the director looks up and waves you over.`
+			`	"Good timing, Captain <first>," he sings, "we have a team of researchers on <planet> who have requested a broken jump drive. Could you deliver one to them?"`
+			choice
+				`	(Maybe later.)`
+					goto refuse
+				`	(Yes.)`
+			`	"Excellent. Your finds have spurred discussion and excitement in various quarters. We have had teams researching jump drives for about at least a century now without a lot of success. I'm told they are rather intricate, but your finds are opening new avenues of investigation. That being said, I'll notify the team on <planet> to expect you."`
+				accept
+			label refuse
+			`	He looks disappointed, and taps something on his screen. "OK, I'll file that request away for now. Come back if you change your mind."`
+				defer
+	on complete
+		conversation
+			`The team waiting for you on <planet> is definitely eager to begin. You barely have the hatch open before they are chivying their camel-pulled wagon up the ramp. They load the broken jump drive surprisingly fast despite their delicate movements.`
+			`	"This is an exciting day, Captain <first>! Countless theories may be proven or sent back for more work based on the testing we can do with this!"`
```

Comment:
"sent back for more work" is not a good phrase for this; you can't really "send back" a theory, since it isn't a physical item. Probably "refined" or "discarded"

Reply:
True, but "sending something back for more work" is a fairly common phrase for telling someone that they need to do more work on it. People routinely "send back" all manner of documents, projects, code, etc that are effectively just ideas and data.

From the dialog, this researcher is obviously of the mindset that no theory is ever truly discarded, simply put back in the fires of the mind for further refinement. Hmmm...

changed to `Countless theories may be proven or put back into the fires of refinement based on the testing we can do with this!`